assets/javascripts/modules/analytics/setup.js

```diff
@@ -1,21 +1,28 @@
-define(['modules/analytics/ga',
-        'modules/analytics/ophan',
-        // 'modules/analytics/thirdPartyTracking'
-], function (ga,
-             ophan,
-            //  thirdPartyTracking
-             ) {
+define([
+    'modules/analytics/ga',
+    'modules/analytics/ophan',
+    'modules/analytics/thirdPartyTracking'
+], function (
+    ga,
+    ophan,
+    thirdPartyTracking
+) {
     'use strict';
 
     function init() {
         ophan.init();
 
-        // GA prefers Ophan to have bootstrapped and set window.guardian state, so load it after Ophan has loaded
-        ophan.loaded.finally(ga.init);
-
-        // thirdPartyTracking.thirdPartyTrackingEnabled().then(thirdPartyTrackingEnabled => {
-            // As this site is mostly deprecated, we've removed all 3rd party trackers.
-        // });
+        /**
+         * GA prefers Ophan to have bootstrapped and set window.guardian state,
+         * so load it after Ophan has loaded if thirdPartyTrackingEnabled.
+        */
+        ophan.loaded.finally(() => {
+            thirdPartyTracking.thirdPartyTrackingEnabled().then(thirdPartyTrackingEnabled => {
```

Comment:
If this wasn't a deprecated site I'd suggest some opportunistic refactoring to rename parts of this API!